src/services/EcoBridge/OmniBridge/subgraph/tokens.ts

```diff
@@ -0,0 +1,25 @@
+import { gql } from 'graphql-request'
+
+export const homeTokensQuery = gql`
+  query homeTokens {
+    tokens(where: { homeAddress_contains: "0x" }, first: 1000) {
+      chainId: homeChainId
+      address: homeAddress
+      name: homeName
+      symbol
+      decimals
+    }
+  }
+`
+
+export const foreignTokensQuery = gql`
+  query foreignTokens {
+    tokens(where: { foreignAddress_contains: "0x" }, first: 1000) {
+      chainId: foreignChainId
+      address: foreignAddress
+      name: foreignName
+      symbol
+      decimals
+    }
+  }
+`
```

Comment:
Should we had pagination to this queries in the future?

Reply:
Yea, we can consider this. Basically we just fetch tokens from subgraph then we create list and pass  to component.